src/core/account.js

```diff
@@ -43,7 +42,7 @@ function Account(remote, account) {
     if (_.includes(Account.subscribeEvents, type)) {
       if (!self._subs && self._remote._connected) {
         self._remote.requestSubscribe()
-        .addAccount(self._account_id)
+        .addAccount(self._address)
         .broadcast().request();
```

Comment:
Not necessarily part of this pull request but we ought to have error handlers for these requests